addon/components/date-range-picker.js

```diff
@@ -59,6 +65,16 @@ export default Ember.Component.extend({
       drops: this.get('drops'),
       opens: this.get('opens')
     });
+
+    this.$('.daterangepicker-input').on('show.daterangepicker', function(ev, picker) {
```

Comment:
I think the "show.daterangepicker" event was something we added to format the start and end date that are displayed when viewing the "Custom Range" option in the date picker. But I don't think its needed anymore. I did a test and commented out lines 79 - 86 (https://github.com/josemarluedke/ember-cli-daterangepicker/blob/master/addon/components/date-range-picker.js#L79) and everything still looked formatted to be.